assets/data/effects/hook/hook_Clown_seriousTrouble_intro.json

```diff
@@ -0,0 +1,658 @@
+							"nameTag": "bottomRight"
+						}
+					],
+					"textSlots": [
+						{
+							"textSourceFile": "hook/hook_Clown_seriousTrouble_intro",
+							"textKey": "~01~prompt~panel_016~1_hook2",
+							"style": "none",
+							"pointerOwner": "hook2",
+							"padXFraction": 0.538,
+							"padYFraction": 0.024
+						}
+					]
+				}
+			}
+		],
+		"options": [
+			{
+				"id": "yes",
+				"script": [
+					{
+						"class": "comicNode_Panel",
+						"panel": {
+							"size": "half",
+							"bg": "comicBG_forestTrunksSunset.jpg",
+							"actorSlots": [
+								{
+									"role": "hook",
+									"form": "frame-joke",
+									"equipment": {}
+								}
+							],
+							"textSlots": [
+								{
+									"textSourceFile": "hook/hook_Clown_seriousTrouble_intro",
+									"textKey": "~02~choice_yes~panel_001~1_hook",
+									"pointerOwner": "hook",
+									"shrinkToFitText": true
+								}
+							]
+						}
+					}
+				]
+			},
+			{
+				"id": "no",
+				"script": [
+					{
+						"class": "comicNode_Panel",
+						"panel": {
+							"size": "half",
+							"bg": "comicBG_library.jpg",
+							"actorSlots": [
+								{
+									"role": "hook",
+									"form": "frame-skeptical",
+									"equipment": {},
+									"anchor": { "x": 0.4, "y": 0.5 },
+									"size": 0.5,
+									"head": { "rotate": -40.365, "scaleY": -1 }
+								}
+							],
+							"textSlots": [
+								{
+									"textSourceFile": "hook/hook_Clown_seriousTrouble_intro",
+									"textKey": "~03~choice_no~panel_001~1_hook",
+									"pointerOwner": "hook",
+									"shrinkToFitText": true
+								}
+							]
+						}
+					}
+				],
+				"uiCancel": true
+			}
+		]
+	},
+	{
+		"role": "party",
+		"template": "JOB_PARTICIPANT",
+		"promptScript": [
+			{
+				"class": "comicNode_Panel",
+				"panel": {
+					"bg": "comicBG_rocksNight.jpg",
+					"actorSlots": [
+						{ "role": "hook", "form": "frame-grim" }
+					],
+					"textSlots": [
+						{
+							"textSourceFile": "hook/hook_Clown_seriousTrouble_intro",
+							"textKey": "~04~prompt~panel_001~1_hook",
+							"style": "narration",
+							"pointerOwner": "hook"
+						}
+					]
+				}
+			}
+		],
+		"slots": {
+			"min": 2,
+			"max": 3,
+			"requiredRoles": [ "hook", "hook2" ]
+		},
+		"isJobResource": false
+	},
+	{
+		"role": "volunteer",
+		"template": "JOB_PARTICIPANT",
+		"choose": "ANY",
+		"slots": null,
+		"notAlreadyMatchedAs": [ "hook", "hook2" ],
+		"alreadyMatchedAs": [ "party" ],
+		"isJobResource": false
+	}
+],
+"outcomes": [
+	{
+		"class": "IfPlayerChose",
+		"then": {
+			"class": "DoAll",
+			"outcomes": [
+				{
+					"class": "AssignToPlot",
+					"assignRolesToPlot": [ "volunteer", "aspect", "party" ]
+				},
+				{ "class": "Special", "specialAction": "ADVANCE_PLOT" }
+			]
+		}
+	}
+],
+"implications": {
+	"generatedTargets": [
+		{
+			"role": "legend",
+			"npcId": "hookClown_seriousTrouble",
+			"test": "1-hook2.drauven_pc",
```

Comment:
Nitpick: Rather than `drauven_pc`, we could also just use the basic `drauven` tag (which has generally been what I've been keying off of) - I have no idea which method is better, but my thought was that if there was some other way someone could get a Drauv party member who didn't have our `drauven_pc` tag then things would still more or less work as expected